impl/src/main/java/org/ehcache/impl/internal/spi/serialization/DefaultSerializationProvider.java

```diff
@@ -274,16 +303,19 @@ private AbstractProvider(Map<Class<?>, Class<? extends Serializer<?>>> serialize
         throw new RuntimeException(e);
       }
     }
+  }
 
-    @Override
-    public void stop() {
-      // no-op
-    }
+  private static class SerializerTuple<T> {
```

Comment:
This class should really be immutable, but that said I disagree with the way this stuff is being lifecycled, or at least the way the lifecycle is being driven.  Maybe we can have a chat at some point?